patches/chrome-VERSION.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/chrome/VERSION b/chrome/VERSION
-index 90a1da4b7a6d8eb71fbe9812e85004f8798c7985..715880cc83bef57e1d9306647b990f0afe836fd6 100644
```

Comment:
I guess this file should stay untouched

Reply:
wasn't sure if this needs to be committed as a result of patching - happy to remove if @bridiver or @mkarolin can verify

Reply:
That's correct, this patch should not be included.

Reply:
got it - removed it